site-content/source/modules/ROOT/pages/community.adoc

```diff
@@ -331,7 +331,7 @@ https://www.apache.org/theapacheway/index.html[The Apache Way,window=blank]
 
 [openblock, float75 full-800]
 ----------
-Contributors are individuals who contribute patches—source code, documentation, help on mailing lists, website—to Apache projects. While contributors do not have a specific governance role, they are crucial to the project’s success. Read the docs to learn how to {site-url}doc/latest/development/index.html[contribute to Cassandra,window=blank], and review our https://cwiki.apache.org/confluence/display/CASSANDRA/Apache+Cassandra+Project+Governance[governance,window=blank] page to understand how we vote on code contributions.
+Contributors are individuals who contribute patches—source code, documentation, help on mailing lists, website—to Apache projects. While contributors do not have a specific governance role, they are crucial to the project’s success. Read the docs to learn how to {site-url}doc/latest/development/index.html[contribute to Cassandra,window=blank], and review our https://cwiki.apache.org/confluence/display/CASSANDRA/Cassandra+Project+Governance[governance,window=blank] page to understand how we vote on code contributions.
```

Comment:
There’s a merge conflict with this page which needs to be resolved:

```
<<<<<<< trunk
Contributors are individuals who contribute patches—source code, documentation, help on mailing lists, website—to Apache projects. While contributors do not have a specific governance role, they are crucial to the project’s success. Read the docs to learn how to {site-url}doc/latest/development/index.html[contribute to Cassandra,window=blank], and review our https://cwiki.apache.org/confluence/display/CASSANDRA/Cassandra+Project+Governance[governance,window=blank] page to understand how we vote on code contributions.
=======
Contributors are individuals who contribute patches—source code, documentation, help on mailing lists, website—to Apache projects. While contributors do not have a specific governance role, they are crucial to the project’s success. Read the docs to learn how to link:{site-url}_/development/index.html[contribute to Cassandra,window=blank], and review our https://cwiki.apache.org/confluence/display/CASSANDRA/Apache+Cassandra+Project+Governance[governance,window=blank] page to understand how we vote on code contributions.
>>>>>>> trunk
```

It should be this:

```suggestion
Contributors are individuals who contribute patches—source code, documentation, help on mailing lists, website—to Apache projects. While contributors do not have a specific governance role, they are crucial to the project’s success. Read the docs to learn how to xref:/development/index.adoc[contribute to Cassandra,window=blank], and review our https://cwiki.apache.org/confluence/display/CASSANDRA/Cassandra+Project+Governance[governance,window=blank] page to understand how we vote on code contributions.
```